package.json

```diff
@@ -16,16 +19,21 @@
     "jest": "^29.4.1",
     "ts-jest": "^29.0.5",
     "tslib": "^2.5.0",
-    "typescript": "4.2.3"
+    "typescript": "^5.0.4"
   },
   "dependencies": {
```

Comment:
polyfills, webpack, and typescript stuff are all devdeps!

Reply:
Ah good catch on webpack. I would argue the polyfills are regular deps as they're being used at runtime and not just used for compilation